apps/server/src/services/rundown-service/rundownCacheUtils.ts

```diff
@@ -30,7 +31,7 @@ export function getLink(currentIndex: number, rundown: OntimeRundown): OntimeEve
  */
 export function handleLink(
   currentIndex: number,
-  rundown: OntimeRundown,
+  rundown: OntimeRundown | DatabaseOntimeRundown, //We can accept both types as we dont care about delay
```

Comment:
suggestion: same as before, I believe we dont need both `OntimeRundown` and `DatabaseOntimeRundown`
The same is applicable to the other function signatures